stage1/iottymux/iottymux.go

```diff
@@ -410,9 +410,10 @@ func actionIOMux(statusFile string) error {
 	switch logMode {
 	case "k8s-plain":
 		var err error
-		// TODO(lucab): check what should be the target path with Euan
-		logTarget := filepath.Join(pathPrefix, appName, "logfile")
-		logFile, err = os.OpenFile(logTarget, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
+		logDir := os.Getenv("KUBELET_LOG_DIR")
+		logPath := os.Getenv("KUBELET_LOG_PATH")
+		logFullPath := filepath.Join(logDir, logPath)
```

Comment:
This just works because there is an implicit invariant that "kubelet logpath on host == logpath in stage1". This should be either documented somewhere, or the `logDir` part should be dropped from here and scoped instead somewhere under `/rkt` or similar (probably better).

/cc @alban

Reply:
I would prefer the latter,  to choose a fixed path inside stage1 so that the kubelet will not be able to overshadow some other file/directory by mistake.

Then, `--bind=` in `stage1/init/init.go` must be updated accordingly using 2 paths.